src/components/notification/notification.tsx

```diff
@@ -0,0 +1,62 @@
+import { CSSProperties, FC, ReactElement, useEffect } from 'react';
+import { NotificationProps, NotificationType } from '@components/notification/types';
+import styles from './notification.module.scss';
+import classNames from 'classnames/bind';
+import { CloseIcon, ErrorIcon, InfoIcon, SuccessIcon } from '@/components';
+
+const cx = classNames.bind(styles);
+const ERROR_DURATION = 7000;
+const DEFAULT_DURATION = 4000;
+
+export const Notification: FC<NotificationProps> = ({
+  uid,
+  title,
+  onClose,
+  icon = null,
+  type = NotificationType.INFO,
+  duration = DEFAULT_DURATION,
+  description,
+  className,
+}): ReactElement => {
+  const adjustedDuration = type === NotificationType.ERROR ? ERROR_DURATION : duration;
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      onClose(uid);
```

Comment:
Why do we need `uid` here? Is it for compatibility with Service UI logic?

Reply:
Yes, for compatibility with Service UI logic